src/app/shipping/itemsRequireShipping.ts

```diff
@@ -0,0 +1,16 @@
+import { Cart, StoreConfig } from '@bigcommerce/checkout-sdk';
+
+const itemsRequireShipping = (cart: Cart | undefined, config: StoreConfig | undefined) => {
```

Comment:
rather than `Cart | undefined` or `StoreConfig | undefined` you can use the optional param `?`

Also, would be good if you can add unit tests for this function.